src/utils/GalleryLoader.ts

```diff
@@ -0,0 +1,193 @@
+import { GalleryData } from "@src/types/gallery";
+
+const data: GalleryData = {
+  Fundamentals: [
+    {
+      title: "Intro to LOOPS",
+      description: "Generate a random number once",
+      imagePath: "../../assets/appGallery/introToLoops.png",
+      youtubeLink: "",
+      appPath: "randomNumber",
+      relevantNodes: [
+        {
+          name: "LOOP",
+          docs: "https://docs.flojoy.ai/nodes/LOGIC_GATES/LOOPS/LOOP/",
```

Comment:
Why are the docs links here if the relevant node badges aren't clickable?
Also you should add more relevant nodes for each app

Reply:
oops I forgot to add it, let me do that